lmdeploy/lite/quantization/awq.py

```diff
@@ -244,6 +254,9 @@ def quant_weights(model,
         if skip_if_contains and skip_if_contains in child_name:
             q_linear = fc
             pack_or_skip = 'skipped'
+        elif 'block_sparse_moe.gate' in name:  # moe
```

Comment:
This is an additional skip patch, considering we already possess the `skip_if _contains` functionality. There is a growing concern that as more Mixtures of Experts (MoEs) are integrated, the skipping branches may become increasingly difficult to maintain.